src/waiters.js

```diff
@@ -3,6 +3,15 @@ const EC = protractor.ExpectedConditions;
 const messageBuilder = require("./constants_and_utils/messageBuilder");
 const utils = require("./constants_and_utils/utils");
 
+const waitForAlertToBePresent = function(
+  timeoutInMilliseconds = utils.timeout.timeoutInMilliseconds
+) {
+  return browser.wait(
```

Comment:
@pedrohyvo, there's a missing parameter after the `timeoutInMilliseconds`, which is the message that should be shown if the alert does not present after the determined timeout.

Take a look at other methods in this file to understand how this is done.

Reply:
Done

Reply:
Done.